cron_jobs/cron.ts

```diff
@@ -54,6 +55,14 @@ const jobs = [
     isActive: config.featureAutoAccept,
     name: 'Auto accept psychologist from designated universities in DS',
   },
+  {
+    cronTime: '0 */6 * * *', // every 6 hours
```

Comment:
I think we can do it once per day, it won't do anything on the last runs of the day.